MCprep_addon/world_tools.py

```diff
@@ -973,6 +974,126 @@ def world_time_update(self, context):
 	return
 
 
+class MCPREP_OT_render_panorama(bpy.types.Operator):
+	"""Render the Panorama images for a texture Pack"""
+	bl_idname = "mcprep.render_panorama"
+	bl_label = "Render Panorama"
+	bl_description = "Render Panorama for texture Pack"
+	bl_options = {'REGISTER', 'UNDO'}
+
+	panorama_resolution = bpy.props.IntProperty(
+		name="Render resolution",
+		description="The resolution of the output images",
+		default=512
+	)
+
+	save_path = bpy.props.StringProperty(
```

Comment:
Per [example here](https://github.com/TheDuckCow/MCprep/blob/55719ce2b1faa3d007f6b2c69c7d0083ac26a90a/MCprep_addon/materials/prep.py#L356) along with according blob filter settings, would be good to stay consistent in path namings, so I'd use `filepath` (even though it's really a directory).